### PR TITLE
Update configure-upgrade-etcd.md to give an example use-case for snap…

### PR DESCRIPTION
**Pull request by ganeshniyer**
…shot using options given by etcdctl

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -269,6 +269,21 @@ If etcd is running on a storage volume that supports backup, such as Amazon
...
 
+### Snapshot using etcdctl options
+
+We can also take the snapshot using various options given by etcdctl. For example 
+
+```shell
+ETCDCTL_API=3 etcdctl --h 
+``` 
+
+will list various options avilable with etcdctl. If you want to take a snapshot by specifying various options available here, you can do so. For example, to take a snapshot by specifying the endpoint, certificates etc is as given below
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:280]:
```suggestion
will list various options available from etcdctl. For example, you can take a snapshot by specifying
the endpoint, certificates etc as shown below:
```

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 2]
```diff
@@ -269,6 +269,21 @@ If etcd is running on a storage volume that supports backup, such as Amazon
...
+
+```shell
+ETCDCTL_API=3 etcdctl --endpoints=[127.0.0.1:2379] --cacert=<trusted-ca-file> --cert=<cert-file> --key=<key-file> snapshot save <backup-file-location>
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:283]:
```suggestion
ETCDCTL_API=3 etcdctl --endpoints=[127.0.0.1:2379] \
  --cacert=trusted-ca-file --cert=cert-file --key=key-file \
  snapshot save backup-file-location
```

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 3]
```diff
@@ -269,6 +269,21 @@ If etcd is running on a storage volume that supports backup, such as Amazon
...
+```
+where, <trusted-ca-file>, <cert-file> and <key-file> can be obtained by looking at the description of the etcd pod.
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:285]:
```suggestion
where `trusted-ca-file`, `cert-file` and `key-file` can be obtained from the description of the etcd Pod.
```

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 4]
```diff
@@ -269,6 +269,24 @@ If etcd is running on a storage volume that supports backup, such as Amazon
...
+```shell
+ETCDCTL_API=3 etcdctl --h 
+``` 
+
+will list various options available from etcdctl. For example, you can take a snapshot by specifying
+the endpoint, certificates etc as shown below:
+
+```shell
+ETCDCTL_API=3 etcdctl --endpoints=[127.0.0.1:2379] \
+  --cacert=trusted-ca-file --cert=cert-file --key=key-file \
```
Comment (reylejano) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:285]:
```suggestion
  --cacert=<path to trusted-ca-file> --cert=<path to cert-file> --key=<path to key-file> \
```

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 5]
```diff
@@ -269,6 +269,24 @@ If etcd is running on a storage volume that supports backup, such as Amazon
...
+  snapshot save backup-file-location
```
Comment (reylejano) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:286]:
```suggestion
  snapshot save <backup-file-location>
```
Angle brackets for placeholders for the backup file location

**Comment by reylejano** [comment]
I think this example fits better in addition to the existing example under the [`Built-in snapshot` heading](en/docs/tasks/administer-cluster/configure-upgrade-etcd.md)

**Review by ganeshniyer — COMMENTED** [review]
Thanks @reylejano for the feedback. If you think the example is appropriate, could you please approve this?

**Pull request merged**